packages/core/src/notification/NotificationService.ts

```diff
@@ -96,8 +96,7 @@ export class NotificationService extends TypedEventEmitter<Events> {
   }
 
   /** Should only be called with a completely new client. */
-  public async initializeNotificationStream(): Promise<string> {
-    const clientId = this.apiClient.clientId;
+  public async initializeNotificationStream(clientId?: string): Promise<string> {
```

Comment:
Does it need to be optional?

Reply:
We can make it required, we call this method only once seems like and we do pass the id, good call 👌